source/includes/examples/_find-the-center.html.md

```diff
@@ -2,7 +2,7 @@
 
 > ![Find the Center Diagram](../images/find-the-center.png)
 
-[**Download the full source code on GitHub**][1] so you can run it yourself on the Bonsai Platform.
+[**Download the full source code on GitHub**][1] if you want to run this simulator locally on the Bonsai Platform.
```

Comment:
Based on my feedback above, I would expect this to be just `[**Download the full source code on GitHub**][1] if you want to run this simulator locally.`